public/app/core/services/keybindingSrv.ts

```diff
@@ -132,16 +132,19 @@ export class KeybindingSrv {
   }
 
   bind(keyArg: string | string[], fn: () => void) {
-    Mousetrap.bind(
-      keyArg,
-      (evt: any) => {
-        evt.preventDefault();
-        evt.stopPropagation();
-        evt.returnValue = false;
-        fn.call(this);
-      },
-      'keydown'
-    );
+    const isInAnIframe = window.location !== window.parent.location;
+    if (!isInAnIframe) {
```

Comment:
Pro-tip: always lean "exits" towards the smaller branch.

For example:
```
  if (isInAnIframe) {
    return;
  }
```

is a lot more readible than the inversion:

```
  if (!isInAnIframe) {
    Mousetrap.bind(
      keyArg,
      (evt: any) => {
        evt.preventDefault();
        evt.stopPropagation();
        evt.returnValue = false;
        fn.call(this);
      },
      'keydown'
    );
  }
```

Fewer lines indented means that the developer has to hold less state in their head when reading the code, as well.

Reply:
Thanks for the tip, applied!